src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -61,4 +61,10 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
     <ProguardConfiguration Include="**\proguard.txt" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <AndroidPackagingOptionsExclude Include="DebugProbesKt.bin" />
+    <AndroidPackagingOptionsExclude Include="$([MSBuild]::Escape('*.kotlin*'))" />
```

Comment:
Given that we now know that `*.kotlin*` is problematic, are we sure we want to continue using it by default?  Or is it that `*.kotlin*`, *in combination with* `@(AndroidPackagingOptionsInclude)` of `*.kotlin_builtins`, means that we're good now?

Reply:
Its the combination which means we are ok now. 
I'm not sure changing to `*.kotlin_m*` would end up including more files than we needed.